src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Corrupted_Native_x86_VS2013_Default.exe.sarif

```diff
@@ -38,18 +38,6 @@
           "executionSuccessful": false
         }
       ],
-      "artifacts": [
```

Comment:
>artifacts [](http://example.com/codeflow?start=7&length=9)

We will only emit an artifact if we have a result.

Reply:
There should be an artifact entry for every artifact that is explicitly referenced. There's not a good reason to arbitrarily restrict this to results. We've lost important information here.